holidays/countries/singapore.py

```diff
@@ -37,6 +37,12 @@
 
 class Singapore(HolidayBase):
     country = "SG"
+    special_holidays = {
+        # SG50 Public holiday
+        # Announced on 14 March 2015
+        # https://www.mom.gov.sg/newsroom/press-releases/2015/sg50-public-holiday-on-7-august-2015
+        2015: ((AUG, 7, "SG50 Public Holiday"),)
+    }
```

Comment:
There are Polling Day dates also

Reply:
It's great to have another pair of eyes! Thank you for reviewing!

Reply:
Thank you guys well done!